mitosheet/mitosheet/mito_dash/v1/mito_callback.py

```diff
@@ -1,28 +1,24 @@
-import gc
-from io import StringIO
-import json
-import time
-from queue import Queue
-from typing import Any, Callable, Dict, List, Optional, Union, Tuple
+from typing import Optional
 
-from mitosheet.mito_dash.v1.spreadsheet import Spreadsheet, WRONG_CALLBACK_ERROR_MESSAGE
+from mitosheet.mito_dash.v1.spreadsheet import Spreadsheet, WRONG_CALLBACK_ERROR_MESSAGE, ID_TYPE
 
 WRONG_CALLBACK_ERROR_MESSAGE_FIRST_LINE = WRONG_CALLBACK_ERROR_MESSAGE.split('\n')[0]
 
 try:
 
-    from dash import callback
+    from dash import callback, Dash, Output, Input, MATCH, State
+    from dash.exceptions import PreventUpdate
+
         
-    def get_component_with_id(id: str) -> Optional[Spreadsheet]:
-        components = [
-            obj for obj in gc.get_objects()
-            if isinstance(obj, Spreadsheet) and getattr(obj, 'mito_id', None) == id
-        ]
+    def get_component_with_mito_id(mito_id: str, session_key: str) -> Optional[Spreadsheet]:
+        """
+        Mito does not work statelessly currently. To work around this limitation of our 
```

Comment:
Move this to the `get instance` code. Document this entire thing in the Spreadsheet component.